### PR TITLE
[INFRA-2928] Assign issue trackers

### PR DESCRIPTION
**Pull request by daniel-beck**
This assigns issue tracker metadata to plugins tracking issues in Jira, in GitHub, and both.

I put the text above `paths` to reduce the potential for conflicts in concurrent changes.

Some individual commits clean up data or otherwise implement manual changes; the 'large' commits were all tool-implemented.

Supersedes #1793. Related to https://github.com/jenkins-infra/repository-permissions-updater/pull/1806, https://github.com/jenkins-infra/repository-permissions-updater/pull/1878, https://github.com/jenkins-infra/update-center2/pull/492, and https://github.com/jenkinsci/jenkins/pull/5358.

### CODE REVIEW

**Comment by daniel-beck** [comment]
https://ci.jenkins.io/job/Infra/job/repository-permissions-updater/job/PR-1884/lastSuccessfulBuild/artifact/json/issues.index.json 🎉

**Review by timja — APPROVED** [review]
scrolled, part of the way through, browser can't handle it though, super laggy =/

**Review by halkeye — APPROVED** [review]
lgtm

Do you have a script that generated this all? Do you want to commit it incase we ever need something like it again (unlikely but it'll be a small useful reference)

**Comment by daniel-beck** [comment]
> Do you have a script that generated this all? Do you want to commit it incase we ever need something like it again (unlikely but it'll be a small useful reference)

Just some hacked together nonsense: https://gist.github.com/daniel-beck/cf56e1857003e091c93be5adeca0f411

The real work was in the input, which was just copied from a giant Google doc. To create that, I basically used your old groovy script but replaced the "write to file" parts with "write to std out" and copied that over, then did spreadsheet things to it (and continue to do that, for upcoming PRs).

**Review by oleg-nenashev — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
We will need to document how to remove the link so that this catalog does not contain misleading entries when plugin maintainers move to GitHub Issues

File: permissions/plugin-artifactory.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 name: "artifactory"
 github: "jenkinsci/artifactory-plugin"
+issues:
+- jira: '15689' # artifactory-plugin
```
Comment (oleg-nenashev) [inline permissions/plugin-artifactory.yml:5]: FTR they handler only submissions to their own tracker.

Comment (daniel-beck) [reply permissions/plugin-artifactory.yml:5]: 🤷

**Review by markyjackson-taulia — APPROVED** [review]

**Review by yJunS — APPROVED** [review]

**Comment by timja** [comment]
any reason not to merge this?

**Comment by daniel-beck** [comment]
> any reason not to merge this?

No strong opinion. I want to hold the core change exposing this to admins for a bit longer to minimize merge conflicts with maintainers setting up issue trackers for their own stuff while I'm still initializing the data, but we can probably proceed with this.

**Comment by daniel-beck** [comment]
I rebased the first commit on top of current master to be able to add further declarations on top of the current state, mostly so I could painlessly incorporate some Azure plugins that were previously assigned Jira but now are in GH.

I'll use this PR to finish this first drop of initial definitions, hence on-holding.

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
@daniel-beck ready to ship?

**Comment by daniel-beck** [comment]
> ready to ship?

Would like an opportunity to check the generated content against reality 😃 I expect this to be done later today or in the next few days.

**Comment by daniel-beck** [comment]
I knew there was something weird: https://github.com/jenkins-infra/repository-permissions-updater/pull/1884/commits/6ce8ba219ef20d97df790e0280f7a769d65169e2 got all of the preferences wrong 😭

**Comment by daniel-beck** [comment]
I am happy with the metadata right now. 😃

**Pull request merged**